f8a_report/report_helper.py

```diff
@@ -673,6 +682,112 @@ def normalize_ingestion_data(self, start_date, end_date, ingestion_data, frequen
             logger.exception('Unable to store the report on S3. Reason: %r' % e)
         return template
 
+    def retrieve_sentry_logs(self, start_date, end_date, frequency='daily'):
```

Comment:
@animuk let's not overpopulate the `class ReportHelper` with unrelated methods. Please create a new file and a new class to include these new methods. This will help to keep the maintainability of the code in check.

Because of this code change, the maintainability of the report_helper.py has reduced to a borderline value which will definitely cause build failures in the future.

Reply:
@samuzzal-choudhury  moved to separate module. changes taken care